test/unicode/utf8.jl

```diff
@@ -1,17 +1,5 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-## Test for CESU-8 sequences
-
-let ch = 0x10000
-    for hichar = 0xd800:0xdbff
-        for lochar = 0xdc00:0xdfff
-            @test convert(String, String(Char[hichar, lochar]).data) == string(Char(ch))
```

Comment:
Out of curiosity, what's the rationale behind removing the constructor for a `String` from an array of `Char`s? To me that kind of construction seems like a very standard, reasonable thing to do. Or am I missing something?